### ISSUE TITLE
Scroll-Bar in JSON-Modul und fertige Zeilen

### ISSUE THREAD

**Issue by Harry423** [Zefau/ioBroker.jarvis#736]
Ich möchte gerne eine History-Element mit dem JSON-Modul anzeigen lassen. 
Dazu habe ich zwei Fragen:

1. Ich möchte die letzten 200 Datensätze sehen, kann aber nicht in der Anzeige scrollen, wie kann ich das einstellen, oder ist das ein Feature?
2. Ich würde auch gern für die bessere Lesbarkeit die Zeilen abwechselnd einfärben (eigentlich nur jede 2.) ist ich das in dem Modul irgendwo definieren?

Danke.

**Comment by Zefau** [comment]
Introduced with v3

**Comment by mcuiobroker** [comment]
Man kann zwar in den EInstellungen die maximale Anzahl der anzuzeigenden Einträge angeben, aber nicht gleichzeitig, die Einstellung für "ALL" angeben.
![image](https://user-images.githubusercontent.com/84818934/132384534-984ac3f5-914b-4dfb-a4d6-3add0a17d6b0.png)
Denn für "ALL" sofort anzeigen muss man maxAnz auf 0 setzen.

**Comment by mcuiobroker** [comment]
Wie verhält sich denn mit der Sortierung, wird dort immer die ganze Tabelle berücksichtigt und dann erst die eingestellte Anzahl der anzuzeigenden Elemente angezeigt?

**Comment by Zefau** [comment]
Ja, wird immer komplett sortiert. Muss ja auch, da es eine Seitennavigation gibt.

> Man kann zwar in den EInstellungen die maximale Anzahl der anzuzeigenden Einträge angeben, aber nicht gleichzeitig, die Einstellung für "ALL" angeben.
image
Denn für "ALL" sofort anzeigen muss man maxAnz auf 0 setzen.

Richtig. Ist das ein Problem? Was fehlt?

**Comment by mcuiobroker** [comment]
Ich möchte immer "all" Einträge sehen, aber von allen nur die ersten 50? Wie kann das einstellen?

**Comment by Zefau** [comment]
`rowsPerPage` auf 50 und `max` auf 0
